### DISCUSSION TITLE
auto columns not populated / deleted rows reappearing / and more

### DISCUSSION THREAD

**Discussion by projestra** [Budibase/budibase#2132] [Help]
Dear all,

when I came across budibase a couple of days ago I was excited by the whole concept. Brilliant - many thanks! 

I've installed budibase as instructed in die manuals via the CLI on an old iMac running Ubuntu 20.04.1. All components from node.js to docker where installed in their most current version. So far so good ... up and running smoothly.

Actually everything woks fine for as long as I work from a single user (which in my case is the budibase admin user). As soon as data rows are added from a second user account, database behaviour becomes very strange (to be on the safe side, I even ditched the whole install incl. containers and volumes, of course but the behaviour stays the same). 

As this is best illustrated with a series of screenshots, I have attached a PDF illustrating the system behaviour. I don't have the faintest idea what may be going on here and would greatly appreciate some insight.

Many thanks in advance and kind regards,
Christian
[The Problem.pdf](https://github.com/Budibase/budibase/files/6885942/The.Problem.pdf)

**Comment by GuilhermeHirsch** [comment]
Hi! How are you, @projestra? Can you tell me which version of Budibase is running?

**Reply by projestra** [reply]
Sure :-) - it's version 0.9.82.
Thank you!

**Comment by shogunpurple** [comment]
@projestra thanks a lot for the detailed bug report. This is great. I'm going to create an issue from this discussion and try to recreate - I will keep you updated on the progress. It seems like something could possibly be wrong with our auto column generation across several users.

**Reply by projestra** [reply]
Thanks a lot! Looking forward to your findings.
Best, Christian
